src/avif.c

```diff
@@ -134,7 +134,19 @@ void avifImageSetDefaults(avifImage * image)
 
 avifImage * avifImageCreate(uint32_t width, uint32_t height, uint32_t depth, avifPixelFormat yuvFormat)
 {
+    // width and height are checked by avifImageAllocatePlanes().
```

Comment:
The callers of `avifImageCreate()` may not call `avifImageAllocatePlanes()`. They may point the `avifImage` directly to existing Y, U, V, alpha buffers. (I wonder if libavif has an example that does this.)

So this function could perform basic validation such as disallowing zero width and height.

UPDATE: I see zero width, height, and depth must be allowed because they are used when creating an empty image (line 160). So perhaps this comment can just say that width and height are checked when they are actually used. Alternatively we can document that this function doesn't validate its input parameters (which is the current behavior).